# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../sfdisk/sfdisk.005.no-llseek-fix.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Imported from buildroot
http://git.uclibc.org/buildroot/tree/package/sfdisk

--- sfdisk_vanilla/sfdisk.c	2004-01-13 13:03:11.000000000 +0000
+++ sfdisk_llseek-fix/sfdisk.c	2008-04-17 08:42:35.000000000 +0000
@@ -134,9 +137,17 @@
  * Note: we use 512-byte sectors here, irrespective of the hardware ss.
  */
 #if defined(__linux__) && !defined (__alpha__) && !defined (__ia64__) && !defined (__x86_64__) && !defined (__s390x__)
+#if defined(_llseek)
 static
 _syscall5(int,  _llseek,  unsigned int,  fd, ulong, hi, ulong, lo,
        loff_t *, res, unsigned int, wh);
+#else
+static int _llseek (unsigned int fd, unsigned long oh,
+		unsigned long ol, long long *result,
+		unsigned int origin) {
+	return syscall (__NR__llseek, fd, oh, ol, result, origin);
+#endif
+}
 #endif
 
 static int
